devnotes/undocumented_registers.md

```diff
@@ -0,0 +1,1062 @@
+# What this document is about?
+This document is an attempt to gather information about undocumented registers
```

Comment:
@IgorBagnucki This document is nor perfectly clear for me at a first glance. You are saying that it gathers information about registers which:
- either are undocumented
- wither work differently than described

Which ones are undocumented?
Which ones work differently than described?
If they work differently than described - what is the description (and where) vs actual operation?